crates/sound/src/vhu_sound.rs

```diff
@@ -2,17 +2,20 @@
 
 use crate::audio_backends::{alloc_audio_backend, AudioBackend};
 use crate::virtio_sound::*;
-use crate::{Error, Result, SoundConfig};
-
+use crate::{Error, SoundConfig};
```

Comment:
in `rust-vmm` they usually organize the `use` section in this way:

```
use std::...
use std::mem::..

use something::...

use crate::...
```

Leaving a blank line to separate each section.

I think the section was in a wrong order before this commit (my bad), so feel free to don't fix here, but please leave the space between them.

Reply:
ok, noted. Thanks :)